portal/static/js/gil.js

```diff
@@ -783,68 +880,72 @@ module.exports = OrgTool = (function() {
                         item.id +'"  ' +  (_isTopLevel ? (' data-parent-id="'+_parentOrgId+'"  data-parent-name="' + _parentOrg.name + '" ') : "") + '/><span>'+
                         i18next.t(item.name) +
                         '</span></label>';
-                    };
+                    }
 
                     childClinic += '</div>';
 
                     if ($("#" + _parentOrgId + "_container").length > 0) $("#" + _parentOrgId + "_container").append(childClinic);
-                    else $("#fillOrgs").append(childClinic);
+                    else {
+                      $("#fillOrgs").append(childClinic);
+                    }
 
                 });
             };
 
-            if (parentOrgsCt > 0 && orgsList[org].isTopLevel) $("#fillOrgs").append("<span class='divider'>&nbsp;</span>");
+            if (parentOrgsCt > 0 && orgsList[org].isTopLevel) {
+              $("#fillOrgs").append("<span class='divider'>&nbsp;</span>");
+            }
         };
     };
     this.handleNoOrgs = function (userId) {
         $(".intervention-link, a.decision-support-link").each(function() {
           var dm = /decision\s?support/gi;
           if (dm.test($(this).text()) || $(this).hasClass("decision-support-link")) {
-              var hasSet = (typeof sessionStorage != "undefined") && sessionStorage.getItem("noOrgModalViewed");
-              //allow modal to show once once action has been taken
-              if (hasSet) return false;
-              var self = this;
-              $.ajax ({
-                  type: "GET",
-                  url: '/api/demographics/' + userId,
-                  async: false
-              }).done(function(data) {
-                  //console.log(data)
-                  if (data && data.careProvider) {
-                      $.each(data.careProvider,function(i,val){
-                          var orgID = val.reference.split("/").pop();
-                          if (parseInt(orgID) === 0) {
-                            $(self).removeAttr("href");
-                            $(self).on("click", function() {
-                              $("figure.js-close-nav").trigger("click");
-                              setTimeout('$("#modal-org").modal("show");', 0);
-                            });
-                            IO.clearSession("decision-support");
-                          };
-                      });
-                  };
-              }).fail(function() {
+            var hasSet = (typeof sessionStorage != "undefined") && sessionStorage.getItem("noOrgModalViewed");
+            //allow modal to show once once action has been taken
+            if (hasSet) {
+              return false;
+            }
+            var self = this;
+            $.ajax ({
+               type: "GET",
+                url: "/api/demographics/" + userId,
+                async: false
+            }).done(function(data) {
+              //console.log(data)
```

Comment:
Could you remove this, or have it only log if the log level is high enough (ie debugging on or development system)?